src/pykeen/nn/vision/image_cache.py

```diff
@@ -0,0 +1,131 @@
+"""Image utilities for wikidata."""
+
+from __future__ import annotations
+
+import functools
+import logging
+import pathlib
+from collections.abc import Collection, Mapping, Sequence
+from textwrap import dedent
+
+from tqdm.asyncio import tqdm
```

Comment:
Afaik, this one is preferred, e.g., because it takes care of notebooks.

```suggestion
from tqdm.auto import tqdm
```

Reply:
Yeah not sure how that happens